### ISSUE TITLE
tests: nftables/binaryutil has no test files

### ISSUE THREAD

**Issue by greenpau** [google/nftables#107]
nftables/binaryutil has no test files:

```
$ go test ./...
?       github.com/google/nftables/binaryutil   [no test files]
```

References:
* [Encoding Data with the Go Binary Package](https://medium.com/learning-the-go-programming-language/encoding-data-with-the-go-binary-package-42c7c0eb3e73)

**Comment by stapelberg** [comment]
pull requests to add tests welcome

**Comment by greenpau** [comment]
> pull requests to add tests welcome

@stapelberg , will make attempt at it.

**Comment by thediveo** [comment]
s'pose that's done?

**Comment by stapelberg** [comment]
Yeah. Next time, the “fixes issue #107” needs to go into the commit message, not the commit title :)

**Comment by thediveo** [comment]
Ouch, thank you for finally reminding me to not constantly overlook this in the future! 😀
